src/mock-doc/element.ts

```diff
@@ -297,6 +314,25 @@ export class MockSVGElement extends MockElement {
   getTotalLength(): number {
     return 0;
   }
+  getComputedTextLength(): number {
+    return 0;
+  }
+  createSVGPoint() {
+    return {
+      matrixTransform: defaultSVGMatrix,
+      x: 0,
+      y: 0,
+    };
+  }
+  getBBox() {
+    return { x: 0, y: 0, width: 10, height: 10 };
```

Comment:
Can we also return the [`top`, `right`, `bottom` and `left` properties](https://developer.mozilla.org/en-US/docs/Web/API/DOMRect#properties) on the `DOMRect` that's returned here to adhere to the interface a little better?

```suggestion
    return { x: 0, y: 0, width: 10, height: 10, left: 0, top: 0, right: 10, right: 10 };
```